src/libraries/System.Security.Cryptography.Xml/tests/SignedXmlTest.cs

```diff
@@ -1593,6 +1593,7 @@ public void VerifyHMAC_HMACOutputLength_Invalid()
         [Theory]
         [InlineData(false)]
         [InlineData(true)]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/74115")]
```

Comment:
If only `true` is failing, it seems worthwhile to only suppress `true`:

```suggestion
        //[InlineData(true)] [ActiveIssue("https://github.com/dotnet/runtime/issues/74115")]
```